kong/plugins/response-ratelimiting/schema.lua

```diff
@@ -75,6 +75,9 @@ return {
           { redis_port = typedefs.port({ default = 6379 }), },
           { redis_password = { type = "string", len_min = 0, referenceable = true }, },
           { redis_username = { type = "string", referenceable = true }, },
+          { redis_ssl = { type = "boolean", required = true, default = false, }, },
+          { redis_ssl_verify = { type = "boolean", required = true, default = false }, },
+          { redis_server_name = typedefs.sni },
```

Comment:
Why we don't need to change `kong/clustering/compat/removed_fields.lua`? @vm-001

Reply:
Will wait for another PR merged and move the internal version compatibility in this PR.